.gitlab-ci.yml

```diff
@@ -109,26 +109,30 @@ codecov:
   needs:
     - check-std
   variables:
-    # According to https://github.com/mozilla/grcov/blob/master/README.md
-    # we have to use these variables.
+    # Variables partly came from https://github.com/mozilla/grcov/blob/master/README.md
     CARGO_INCREMENTAL:             0
     # If `-Zno-landing-pads` is enabled some some tests start to fail.
```

Comment:
```suggestion
```

Reply:
We can remove this line since `-Zno-landing-pads` has been replaced by `-Cabort=panic`.